### PR TITLE
added login and register forms

### PR DESCRIPTION
**Pull request by Liubko89**
link for "forgot password" in login page will add another time. For now have no idea how the link might work. Also added file vercel.json - for deploying to vercel

### CODE REVIEW

File: src/services/schemas.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import * as Yup from 'yup';
```
Comment (Basicula) [inline src/services/schemas.ts:1]:
```suggestion
import { object as YupObject, string as YupString } from 'yup';
```

File: src/components/RegisterForm/RegisterForm.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import { ErrorMessage, Field, Form, Formik, FormikHelpers } from 'formik';
+import css from './AuthForm.module.css';
+import { registerSchema } from '../../services/schemas';
+import { FcGoogle } from 'react-icons/fc';
+
+const RegisterForm: React.FC = () => {
+  type FormData = {
+    name: string;
+    tel: string;
```
Comment (Basicula) [inline src/components/RegisterForm/RegisterForm.tsx:9]: Please, try to use full name for variables, don't shrink it in future
```suggestion
    phone: string;
```

File: src/components/LoginForm/LoginForm.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+              name="password"
+              placeholder="Enter password"
+              autoComplete="off"
+            />
+            <ErrorMessage
+              className={css.errorMsg}
+              name="password"
+              component="span"
+            />
+          </label>
```
Comment (Basicula) [inline src/components/LoginForm/LoginForm.tsx:61]: You have a total duplication here that can be replaced with simple react component like FormLabel, with argument that are differs here

Comment (Basicula) [reply src/components/LoginForm/LoginForm.tsx:61]: Also same code is duplicated in RegisterForm

Comment (Liubko89) [reply src/components/LoginForm/LoginForm.tsx:61]: the fields are different, they have types, names, placeholders which are not the same

File: src/components/RegisterForm/RegisterForm.tsx (hunk 2 above)
Comment (Liubko89) [reply src/components/RegisterForm/RegisterForm.tsx]: done

File: src/components/LoginForm/LoginForm.tsx (hunk 3 above)
Comment (Basicula) [reply src/components/LoginForm/LoginForm.tsx:61]: So you pass those fields as arguments

**Review by YuliaYaremchuk — APPROVED** [review]
register form is good.

**Comment by Basicula** [comment]
You have two implementations for FormLabel please choose and leave only one.

File: src/services/schemas.ts (hunk 1 above)
Comment (Liubko89) [reply src/services/schemas.ts]: done

**Pull request merged**